statsmodels/tsa/arima/estimators/tests/test_hannan_rissanen.py

```diff
@@ -100,3 +106,238 @@ def test_unbiased_error():
     endog = (np.arange(1000) * 1.0)
     with pytest.raises(ValueError, match='Cannot perform third step'):
         hannan_rissanen(endog, ar_order=1, ma_order=1, unbiased=True)
+
+
+def test_set_default_unbiased():
```

Comment:
Adding this test and `test_set_default_unbiased_with_fixed_params` below to test the added handling of `unbiased` and `fixed_params` args.